frontend/src/plugins/tenantManagement/templates/tenantManagementSidebar.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="sidebar-item-inner">
+  <div class="action-button-group">
```

Comment:
`<div class="separator"></div>` needs to be added here so the spacing matches the other sidebar buttons.